disnake/http.py

```diff
@@ -111,20 +111,15 @@ async def json_or_text(response: aiohttp.ClientResponse) -> Union[Dict[str, Any]
     return text
 
 
-def to_multipart(payload: Dict[str, Any], files: Iterable[File]) -> List[Dict[str, Any]]:
-    # NOTE: this method modifies the provided `payload` and `payload["attachments"]` collections
+def set_attachments(payload: Dict[str, Any], files: Sequence[File]) -> None:
+    """
+    Updates the payload's attachments list based on the provided files
+
+    note: this method modifies the provided `payload` and `payload["attachments"]` collections
```

Comment:
You probably wanted to type \`\`payload\`\` and \`\`payload["attachments"]\`\`, same thing in other docstrings.
Doesn't matter though, these functions aren't public anyways.